.changeset/chilled-boats-thank.md

```
@@ -0,0 +1,7 @@
+---
+"@assistant-ui/react-markdown": patch
+"@assistant-ui/react-trieve": patch
+"@assistant-ui/react": patch
+---
+
+fix: cjs builds
```

Markdownlint (0.35.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)